content/docs/tasks/security/authn-policy/index.md

```diff
@@ -730,7 +730,7 @@ EOF
 {{< /text >}}
 
 {{< tip >}}
-Use `istio create` if the `jwt-example` policy hasn't been submitted.
+Use `kubectl create` if the `jwt-example` policy hasn't been submitted.
```

Comment:
A better fix would be to remove this tip completely (delete lines 732-735), because `kubectl apply` works in all cases (it creates if necessary).

Reply:
I agree too.